rust/src/dcerpc/dcerpc.rs

```diff
@@ -1197,7 +1205,15 @@ pub extern "C" fn rs_dcerpc_parse_response(
     if input_len > 0 {
         if input != std::ptr::null_mut() {
             let buf = build_slice!(input, input_len as usize);
-            return state.handle_input_data(buf, core::STREAM_TOCLIENT);
+            let ret = state.handle_input_data(buf, core::STREAM_TOCLIENT);
+            let tx_id = state.tx_id - 1;
```

Comment:
Is this correct? I would expect a tx to be created on the request side, not the response. So here I would expect the `state.tx_id` to stay the same.